### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by Eshy10**
## Readme File containing instructions on how to play the game
- Added a Readme File
- The Readme File contains instructions on how to play and run the game

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+- The players are required to input numbers between 0 - 8, to pick a position in the board.
+- If the player pick a position that is occupied, the player will be required to pick another position.
+- The first player to get identical letter in the row or column or left and right diagonal wins the game.
+- If no identical letter occurs and the board is full, the game will end in a draw
+
+## How to run the game in your terminal
+
+- Run this command line: bin/main.rb
+
+**This is an example of how you may give instructions on setting up your project locally.**
```
Comment (euqueme) [inline README.md]: - [x] kindly remove line #22 because it belongs to the Readme template you used but not to your project

File: README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Install ruby on your local machine
+
+### Setup
+
+- Clone the repository on your local machine
```
Comment (euqueme) [inline README.md]: - [x] Final user's don't need to clone the repo they can just download it, let's tell them how

File: README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+- cd into the folder, find the bubble_sort file and open it
```
Comment (euqueme) [inline README.md]: - [x] bubble_sort is not part of your project, let's change this line

File: README.md [hunk 4]
```diff
@@ -1,2 +1,74 @@
...
+
+### Install
+
+- Install Ruby (most recents versions)
+- Install VSCode or any code editor you like
+- Add gem 'rubocop' to Gemfile (not sure how to use Gemfile? Read <a href="https://bundler.io/v1.15/guides/bundler_setup.html">this</a>).
+- Run bundle install.
```
Comment (euqueme) [inline README.md]: - [x] Final users don't need VSCode, or rubocop they only need to have ruby installed in their computers so let's put these instructions in a section for developers or contributors

File: README.md [hunk 5]
```diff
@@ -1,2 +1,74 @@
...
+
+### Run tests
+
+- Add a copy of <a href="https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml">.stickler.yml</a> and <a href="https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml">.rubocop.yml</a> to root directory.
+- Run rubocop to fix linters errors
```
Comment (euqueme) [inline README.md]: - [x] let's also put the Run test section in a section for Developers or Contributors within your Readme file

File: README.md [hunk 6]
```diff
@@ -1,2 +1,74 @@
...
+
+- Github: [@githubhandle](https://github.com/vzdrizhni)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- Linkedin: [linkedin](https://twitter.com/metabruta)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page]().
```
Comment (euqueme) [inline README.md]: - [x] right now the issues page link leads to an empty page, add the URL within the parenthesis to the issues page link or remove it from the Readme file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Eshy10 and @vzdrizhni :balloon: :lollipop: 

### You've done a great job with this project overall and using a good template for your Readme file :star: :star: there's a couple things to change so please address the comments I've left in your code :point_down: 

[OPTIONAL]

#### Behaviour

- [x] Your game lets players have empty names or the same name and when they do the game is giving them the same mark('X'), so let's add a validation for this to avoid ambiguity preventing the users to have the same name or set a default name when user's don't provide one. this wasn't pointed out in previous reviews but I strongly recommend to fix it:

image

- [x] Your game also stops working whenever a user try to pick an already taken cell and then try to pick the number 9 I think you also need a validation so the game doesn't stop working. again this wasn't pointed out in the last milestone but I strongly recommend to fix it:

image

### That's it, team! :raised_hands: 

### Please :pray: do the changes in your Readme file (and the optional changes or not) :woman_technologist: :man_technologist: and submit back for a code review when you are done :memo: 

Happy Coding! :rocket: 

Any questions? ask me on slak **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good Job
Your README looks good but we need to add rules for the game before we can merge.
- [ ] README file should explain the rules of the game. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
- [ ] add this change and send for another review

### Martin Afani

**Comment by Eshy10** [comment]
> ## Good Job
> Your README looks good but we need to add rules for the game before we can merge.
> 
> * [ ]  README file should explain the rules of the game. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
> * [ ]  add this change and send for another review
> 
> ### Martin Afani

**Comment by Eshy10** [comment]
Instructions are in the readme file, please kindly check

**Review by whiz25 — APPROVED** [review]
## Hey @Eshy10  and @vzdrizhni 
Merge and proceed. Your README looks good.

### Martin Afani

**Pull request merged**